lib/notification/apsProperties.js

```diff
@@ -83,6 +83,36 @@ module.exports = {
     }
   },
 
+  set relevanceScore(value) {
+    if (typeof value === 'number' || value === undefined) {
+      this.aps['relevance-score'] = value;
+    }
+  },
+
+  set timestamp(value) {
+    if (typeof value === 'number' || value === undefined) {
+      this.aps.timestamp = value;
+    }
+  },
+
+  set staleDate(value) {
+    if (typeof value === 'number' || value === undefined) {
+      this.aps['stale-date'] = value;
+    }
+  },
+
+  set event(value) {
+    if (typeof value === 'string' || value === undefined) {
+      this.aps.event = value;
```

Comment:
Is this supposed to be `events`? The docs here refers to the key as "events": https://developer.apple.com/documentation/activitykit/updating-and-ending-your-live-activity-with-activitykit-push-notifications

Reply:
Good catch!